pallav/fashionTech/pallavBanka/parkingAssignment/Models/Slot.java

```diff
@@ -0,0 +1,21 @@
+package fashionTech.pallavBanka.parkingAssignment;
+
+import java.util.LinkedList;
+
+public class Slot{
+    LinkedList<String> occupied=new LinkedList();
```

Comment:
Use List<String> occupied = new LinkedList();
Try to declare always using interface, rather then concrete implementation